packages/trader/src/sass/app/_common/components/vertical-tabs.scss

```diff
@@ -18,7 +18,7 @@ $MIN_HEIGHT: calc(100vh - #{$HEADER_HEIGHT + $FOOTER_HEIGHT} - #{$BOTTOM_SPACING
         }
     }
     &:not(.vertical-tab--full-screen) {
-        height: 600px;
+        height: 500px;
```

Comment:
I think this `600px` was according to the design, any reason reducing it to 500?

Reply:
![image](https://user-images.githubusercontent.com/3080372/64094065-0fc28580-cd8d-11e9-8d49-b56a4e0f2581.png)

Reply:
@reza-fs it was overlapping in different resolution, i just wanted to have a safer height.

Reply:
:bowing_man: thanks.